wandb/apis/public/artifacts.py

```diff
@@ -412,7 +427,9 @@ def change_type(self, new_type: str) -> None:
         """
         if not self.is_sequence():
             raise ValueError("Artifact collection needs to be a sequence")
-        termlog(f"Changing artifact collection type of " f"{self.type} to {new_type}")
+        termlog(
+            f"Changing artifact collection type of " f"{self._saved_type} to {new_type}"
+        )
```

Comment:
```suggestion
        termlog(
            f"Changing artifact collection type of {self._saved_type} to {new_type}"
        )
```
idk why this wasn't just one joined string in the first place. I imagine it was an autoformat change by `ruff` or `black`.